slurk/views/chat/events.py

```diff
@@ -66,6 +66,14 @@ def bounding_box(payload):
             )
 
 
+@socketio.event
+def keystroke(key):
+    db = current_app.session
+    room = db.query(Room).get(key["room"]) if "room" in key else None
```

Comment:
when would room be None?

Reply:
and does it make sense to log in that case?

Reply:
I'm pretty sure that it shouldn't be possible, but I saw this check in the ```mouse``` event, but you're right, if we cannot match it to a room there is no reason to log it, I'm going to change it